libs/agora/teams/src/lib/team-member-list/team-member-list.component.scss

```
@@ -42,8 +42,15 @@
     margin-bottom: 0;
     text-align: center;
 
-    // a {
-    //   @include link;
-    // }
+    a {
+      color: var(--color-link);
+      cursor: pointer;
+      text-decoration: none;
+
+      &:hover {
+        color: var(--color-link);
+        text-decoration: underline;
+      }
+    }
```

Could we use the imported `mixins` instead so the shared styles are used?

```suggestion
    a {
      @include mixins.link;

      &:hover {
        @include mixins.link-hover;
      }
    }
```